### PR TITLE
[addons/awscsidriver] Bump to GA release

### PR DESCRIPTION
**Pull request by dntosas**
Bump version and manifests of AWS EBS CSI Driver to the first GA release.

Signed-off-by: dntosas <ntosas@gmail.com>

### CODE REVIEW

File: upup/models/cloudup/resources/addons/aws-ebs-csi-driver.addons.k8s.io/k8s-1.17.yaml.template [hunk 1]
```diff
@@ -211,6 +211,45 @@ roleRef:
...
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: ebs-csi-node-binding
+  labels:
+    app.kubernetes.io/name: aws-ebs-csi-driver
+subjects:
+  - kind: ServiceAccount
+    name: ebs-csi-node-sa
+    namespace: default
```
Comment (olemarkus) [inline upup/models/cloudup/resources/addons/aws-ebs-csi-driver.addons.k8s.io/k8s-1.17.yaml.template:224]: Should this be in the default namespace?

Comment (dntosas) [reply upup/models/cloudup/resources/addons/aws-ebs-csi-driver.addons.k8s.io/k8s-1.17.yaml.template:224]: omg, thanks for catching ^

Comment (dntosas) [reply upup/models/cloudup/resources/addons/aws-ebs-csi-driver.addons.k8s.io/k8s-1.17.yaml.template]: fixed

**Review by olemarkus — APPROVED** [review]

**Comment by olemarkus** [comment]
Thanks. I think these can be cherry-picked back to 1.20.

**Pull request merged**